### PR TITLE
fix: fix newly generated attachment not shown in file manager (Issue# 1453)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**

Fixed newly generated attachment not shown in the file manager screen

Issues:

- Issue #1453 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1520.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/store/conversations/conversations.epics.ts [hunk 1]
```diff
@@ -1049,21 +1049,21 @@ const updateMessageEpic: AppEpic = (action$, state$) =>
...
 
       if (attachments) {
-        const attachmentFolders = uniq(
+        const attachmentRootFolders = uniq(
           attachments
             .map(
               (attachment) =>
                 attachment.url &&
-                getFolderIdFromEntityId(decodeURIComponent(attachment.url)),
+                getRootFolderIdFromEntityId(decodeURIComponent(attachment.url)),
```
Comment (IlyaBondar) [inline apps/chat/src/store/conversations/conversations.epics.ts]: Use `getParentFolderIdsFromEntityId` instead to reset statuses for all parent hierarchy

**Review by IlyaBondar — CHANGES_REQUESTED** [review]

File: apps/chat/src/store/conversations/conversations.epics.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/store/conversations/conversations.epics.ts]: Thank you for the suggestion!
Changed to reset all parent hierarchy.

File: apps/chat/src/store/conversations/conversations.epics.ts [hunk 2]
```diff
@@ -1049,21 +1048,23 @@ const updateMessageEpic: AppEpic = (action$, state$) =>
         messages[payload.messageIndex].custom_content?.attachments;
 
       if (attachments) {
-        const attachmentFolders = uniq(
+        const attachmentRootFolders = uniq(
```
Comment (IlyaBondar) [inline apps/chat/src/store/conversations/conversations.epics.ts]:
```suggestion
        const attachmentParentFolders = uniq(
```

Comment (denys-kolomiitsev) [reply apps/chat/src/store/conversations/conversations.epics.ts]: Suggestion commited. Thank you!

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1520.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**